account_move_helper/models/account_account.py

```diff
@@ -33,13 +31,17 @@ def _compute_new_balance(self):
 
     @api.multi
     def _compute_balance(self):
+        move_id = self._context.get('active_id', False)
+        move = self.env['account.move'].browse(move_id)
         for rec in self:
-            # rec.balance = rec._get_balance()
+            domain = [
+                ('account_id', '=', rec.id),
+                ('move_id.state', '=', 'posted'),
+            ]
+            if move:
+                domain.append(('date', '<=', move.date))
             rec.balance = sum(
-                rec.env['account.move.line'].search([
-                    ('account_id', '=', rec.id),
-                    ('move_id.state', '=', 'posted'),
-                ]).mapped('balance'))
+                rec.env['account.move.line'].search(domain).mapped('balance'))
 
     @api.multi
     def _inverse_new_balance(self):
```

Comment:
convert this comments into method docstring

Reply:
this method call another method that raise an UserError.
I am not sure but maybe this could be a problem